cmd/gardener-extension-validator-aws/app/app.go

```diff
@@ -18,18 +18,17 @@ import (
 	"context"
 	"fmt"
 
+	admissioncmd "github.com/gardener/gardener-extension-provider-aws/pkg/admission/cmd"
```

Comment:
I also see that with https://github.com/gardener/gardener-extension-provider-gcp/pull/69, `validator-gcp` is renamed to `admission-gcp`. I just wanted to raise this question here as I guess we want to unify the image (artifact) names. @timuthy , @rfranzke , correct me if I am wrong. We can just clarify the target name that we want (I guess the plan is to have `admission-*` instead of `validator-*`) and then create an issue for such renaming.

Reply:
You're right @ialidzhikov. Renaming to `admission` was intended as these components are going to be used for defaulting (mutation) as well.

Reply:
@timuthy @ialidzhikov  - Should I make the image renaming in this PR then ?

Reply:
You can also do it in subsequent PR. It's "just" to follow the convention. Extension webhooks experienced several reworks in the past and GCP used to be some kind of "role model", since we've started with this repo but unfortunately didn't have capacity to replicate the work to all other extension repos. (Luckily, you're doing it now for AWS :))